diff --git a/out/binDownload.js b/out/binDownload.js
index 6ffda4bbecf188135d84f291475fa529c24e2e93..e6f7534299018079b172660555cc5b50448e8056 100644
--- a/out/binDownload.js
+++ b/out/binDownload.js
@@ -49,7 +49,7 @@ function getBin(cacheKey, url, checksum) {
         return promise;
     }
     console.log('getBin', cacheKey, cacheName, url)
-    promise = doGetBin(cacheName, url, checksum);
+    promise = doGetBin(cacheKey, url, checksum);
     versionToPromise.set(cacheName, promise);
     return promise;
 }
