diff --git a/out/util/packageMetadata.js b/out/util/packageMetadata.js
index 3d006f7aae80f03aa7f0d8229cd7fdf8c15d9d9e..bf4c60bd48408e6cb320c4ee91a16acdbdaa23c4 100644
--- a/out/util/packageMetadata.js
+++ b/out/util/packageMetadata.js
@@ -80,11 +80,26 @@ function checkDependencies(dependencies, errors) {
     if (dependencies == null) {
         return;
     }
-    const updaterVersion = dependencies["electron-updater"];
-    const requiredElectronUpdaterVersion = "4.0.0";
-    if (updaterVersion != null && !versionSatisfies(updaterVersion, `>=${requiredElectronUpdaterVersion}`)) {
-        errors.push(`At least electron-updater ${requiredElectronUpdaterVersion} is recommended by current electron-builder version. Please set electron-updater version to "^${requiredElectronUpdaterVersion}"`);
-    }
+
+    let updaterVersion = dependencies["electron-updater"]
+    if (updaterVersion != null) {
+      // Pick the version of a yarn berry patch syntax
+      // "patch:electron-updater@npm%3A6.4.1#~/.yarn/patches/electron-updater-npm-6.4.1-ef33e6cc39.patch"
+      if (updaterVersion.startsWith('patch:')) {
+        const match = updaterVersion.match(/@npm%3A(.+?)#/)
+        if (match) {
+          updaterVersion = match[1]
+        }
+      }
+  
+      const requiredElectronUpdaterVersion = "4.0.0"
+      if (!versionSatisfies(updaterVersion, `>=${requiredElectronUpdaterVersion}`)) {
+        errors.push(
+          `At least electron-updater ${requiredElectronUpdaterVersion} is recommended by current electron-builder version. Please set electron-updater version to "^${requiredElectronUpdaterVersion}"`
+        )
+      }
+    }
+    
     const swVersion = dependencies["electron-builder-squirrel-windows"];
     if (swVersion != null && !versionSatisfies(swVersion, ">=20.32.0")) {
         errors.push(`At least electron-builder-squirrel-windows 20.32.0 is required by current electron-builder version. Please set electron-builder-squirrel-windows to "^20.32.0"`);
